README.md

```diff
@@ -30,10 +30,13 @@ Countless searches to find an app to solve these problems and failing to find on
 - It's FREE and shall ALWAYS remain FREE!!
 
 
-### Download App
-Download the [Latest Version](https://ganeshrvel.github.io/openmtp/?downloadApp=github&release=stable&platform=mac 'Latest Version')
+### Installation
+* Download the [Latest Version](https://ganeshrvel.github.io/openmtp/?downloadApp=github&release=stable&platform=mac 'Latest Version')
 
-Find the latest *dmg* file from [GitHub Releases](https://github.com/ganeshrvel/openmtp/releases "GitHub Releases")
+* Using Homebrew Cask
+  * ```brew cask install openmtp```
```

Comment:
Need some info:
I haven't worked on brew casks before. 
How is the cask maintained?
How do we point to the latest version of the app in the cask?
Does it require any secret keys to update the cask?

Reply:
First of all, thank you very much for checking my pull request so quickly :-D

> How is the cask maintained?
> How do we point to the latest version of the app in the cask?

* These two questions point to the same answer. People submit pull requests to [homebrew's GitHub repository](https://github.com/Homebrew/homebrew-cask) for daily maintenance. For caks that contain update sources such as appcast in the cask file, homebrew also has robots for automatic updates.Because homebrew only allows download links officially released by software authors such as the software homepage or GitHub Release address, and the entire process is verifiable and open source, it is safe to use homebrew for software installation.This is also a popular way to install software on macOS, similar to the package manager used by Linux

> Does it require any secret keys to update the cask?
* No.Homebrew does not transfer or handle the process of downloading and installing software. It only installs according to the given rb files and instructions. And check the sha256 checksum after downloading to ensure that the downloaded file is correct.

Finally, since I am not a native speaker of English, there may be errors or omissions in the related expressions. If you want to know more about these,I would be happy to explain further :-D